scripts/generate_hashes.py

```diff
@@ -0,0 +1,75 @@
+"""
+This script will output a json document containing, for each ggcanary token, it's name and hash.
+It only depends on the python standard library (python >= 3.7)
+
+
+The output has the following the format:
+```
+[
+  {
+    "name": "canary_token: <token name>",
+    "hash": <hash>
+  },
+  ...
+]
+```
+
+"""
+
+#!/usr/bin/env python3
```

Comment:
This will not work because of the comments above. Shebang must be in the first line.